src/js-api.test.ts

```diff
@@ -93,6 +104,15 @@ describe(`deploy command`, () => {
     delete process.env.AWS_ACCESS_KEY_ID;
     delete process.env.AWS_SECRET_ACCESS_KEY;
 
+    const entryUrl = `/navbar.js`;
+    const s3Key = `navbar/navbar.js`;
+    const deployApiReqInit =
+      baseplateFetchHistory[`/api/orgs/${defaultOrgId}/deployments`];
+    expect(deployApiReqInit.body.changedMicrofrontends[0].entryUrl).toEqual(
```

Comment:
The assertions here test that things still work when the `--autoVersion` flag isn't present.